src/artofillusion/polymesh/PolyMeshViewer.java

```diff
@@ -2,7 +2,7 @@
  *  Copyright (C) 1999-2004 by Peter Eastman (TriMeshViewer.java),
  *  Modifications for Winged Edge Mesh Copyright (C) 2004-2005 by François Guillet
  *  Modifications for mouse buttons Copyright (C) 2019 by Petri Ihalainen
- *
+ *  Changes copyright (C) 2024 by Maksim Khramov
```

Comment:
You need a blank line between the copyright statements and the license declaration, for readability if nothing else.